giraf/giraf.go

```diff
@@ -17,7 +17,7 @@ type Giraf struct {
 	Flag      uint8
 	PosStrand bool
 	Path      *Path
-	Aln       []*cigar.Cigar // current cigar will need to be expanded
+	Aln       []simpleio.ByteCigar// current cigar will need to be expanded
```

Comment:
I am not sure if the simpleio package is in this request so I don't think I can see the data structure.  Would it make sense to put this data structure in the cigar package?